components/crowdfunding-redesign/edit/common.tsx

```diff
@@ -0,0 +1,236 @@
+              min: props.min,
+              max: props.max,
+              required: props.required,
+              error: hasError,
+              placeholder,
+            },
+            value => value !== undefined,
+          ),
+        };
+        if (
+          'required' in fieldAttributes &&
+          !fieldAttributes.required &&
+          meta.error &&
+          meta.error === intl.formatMessage(RICH_ERROR_MESSAGES.requiredValue)
+        ) {
+          fieldAttributes.required = true;
+        }
+        return (
+          <div className="flex w-full flex-col gap-1">
+            {label && <Label className="text-base leading-normal">{label}</Label>}
+            {hint && <p className="text-sm text-muted-foreground">{hint}</p>}
+            {children ? children({ form, meta, field: fieldAttributes }) : <Input {...fieldAttributes} />}
+            {hasError && showError && (
+              <p className="text-xs text-red-500">
+                {isOCError(meta.error) ? formatFormErrorMessage(intl, meta.error) : meta.error}
+              </p>
+            )}
+          </div>
+        );
+      }}
+    </Field>
+  );
+}
+
+export function ColumnSection({ title, description, children }) {
+  return (
+    <div className="grid grid-cols-12 gap-4">
+      <div className="col-span-full md:col-span-4">
+        <div className="text-lg font-medium">{title}</div>
+        <p className="text-sm text-muted-foreground">{description}</p>
+      </div>
+      <div className="col-span-full md:col-span-8 md:col-start-6">
+        <div className="flex flex-col gap-4">{children}</div>
+      </div>
+    </div>
+  );
+}
+
+function ColorPicker({ value, onChange }) {
+  // Selected colors from tailwind palette
+  const options = [
+    '#ef4444', // red-500
+    '#f97316', // orange-500
+    '#f59e0b', // amber-500
+    '#16a34a', // green-500
+    '#14b8a6', // teal-500
+    '#0284c7', // sky-600
+    '#1d4ed8', // blue-700
+    '#4f46e5', // indigo-600
+    '#c026d3', // fuchsia-600
+    '#db2777', // pink-600
+    '#475569', // slate-600
+    '#020617', // slate-950
+  ];
+  return (
+    <div>
+      <Popover>
+        <PopoverTrigger>
+          <Button variant="outline" className="gap-2">
+            <div className="size-4 rounded bg-primary" style={{ backgroundColor: value }} />
+            {value}
+          </Button>
+        </PopoverTrigger>
+        <PopoverContent align="start" className="w-auto">
+          <div className="flex flex-col gap-2">
+            <div className="flex flex-wrap gap-1.5">
+              {options.map(color => (
+                <button
+                  onClick={() => onChange(color)}
+                  key={color}
+                  className="size-6 rounded-md"
+                  style={{ backgroundColor: color }}
+                />
+              ))}
+            </div>
+            <InputGroup prepend="#" value={value.replace('#', '')} onChange={e => onChange(`#${e.target.value}`)} />
```

Comment:
Suggestion: add an input `type=color` next to it, to let people select a different color using native APIs

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/9f71df21-3d2a-48f2-b9fd-46c52cc331f8)

If you want to try it:

```suggestion
            <div class="flex gap-2 items-center">
              <input type="color" value={value} class="rounded-full w-8 h-8 cursor-pointer">
              <InputGroup prepend="#" value={value.replace('#', '')} onChange={e => onChange(`#${e.target.value}`)} />
            </div>
```

Reply:
Nice, added something like this, with some modified styles:

<img width="464" alt="image" src="https://github.com/opencollective/opencollective-frontend/assets/1552194/c4c5edef-6cf0-4e06-99ad-286909c800eb">